OracleGoldenGate/README.md

```diff
@@ -0,0 +1,98 @@
+Oracle GoldenGate on Docker
+===============
+Sample Docker build files to provide an installation of Oracle GoldenGate for DevOps users. For more information about Oracle GoldenGate please see the [Oracle GoldenGate On-line Documentation](https://docs.oracle.com/goldengate/c1221/gg-winux/index.html).
+
+## How to build and run
+This project provides a Dockerfile tested with:
+ * Oracle GoldenGate 12c Release 2 (12.2.0.1)
+ * Oracle GoldenGate Standard Edition 12c Release 3 (12.3.0.1)
+ * Oracle GoldenGate Microservices Architecture 12c Release 3 (12.3.0.1)
+
+To build the images, use the [buildDockerImage.sh](buildDockerImage.sh) script.
+
+### Building Oracle GoldenGate Docker Images
+**IMPORTANT:** You must download the installation binaries of Oracle GoldenGate. You only need to provide the binaries for the version you plan to install. The binaries can be downloaded from the [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html). You also must have Internet connectivity when building the Docker image for the package manager. Note that you must not uncompress the Oracle GoldenGate binaries. The script will handle that for you and fail if you uncompress them manually!
+
+Once you have downloaded the Oracle GoldenGate software, run the **`buildDockerImage.sh`** script:
```

Comment:
You can't rely on users to use your `buildDockerImage.sh` script. This is provided for convenience but must not be mandatory. Please provide instructions on how to prepare the binaries for a manual build.

This is to ensure that orchestration or CI/CD systems can be used to build the image without needing to run scripts first.